.github/workflows/icount-bench.yml

```diff
@@ -0,0 +1,34 @@
+name: icount bench
+on: [pull_request]
+
+jobs:
+  icount-benchmarks:
+    name: Run icount benchmarks
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install valgrind
+        run: sudo apt install -y valgrind
+
+      - name: Install stable toolchain
+        uses: dtolnay/rust-toolchain@nightly
```

Comment:
The comment here says stable, but it's using nightly. Could one of the two be updated so they're in-sync?

Reply:
Thanks for that, let's use stable so we have one less variable to account for noise